starters/vue3-apollo-quasar/src/pages/IndexPage.vue

```diff
@@ -19,3 +19,14 @@ export default defineComponent({
   name: 'IndexPage',
 });
 </script>
+
+
+<style lang="scss" scoped>
+main {
+  min-height: unset!important;
+  margin-top: 3rem;
```

Comment:
use utility classes to have margin values reusable

Reply:
We might not need that as its not going to be large. Its just a starter kit

Reply:
ok. got it now.

Reply:
Thanks